pre_commit_hooks/php_lint.sh

```diff
@@ -1,6 +1,28 @@
-#!/bin/bash
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+export PATH=$PATH:/usr/local/bin
+
+if ! which php-cs-fixer &>/dev/null; then
+    >&2 echo 'php-cs-fixer command not found'
+fi
+if [ `uname` = "Darwin" ]; then
+    if [ -z "$(command -v brew)" ]; then
+        exit 1
+    elif [ -z "$(command -v php-cs-fixer)" ]; then
+            brew install php-cs-fixer
+    fi
+elif [ `uname` = "Linux" ]; then
+    if [ -z "$(command -v php-cs-fixer)" ]; then
+        sudo curl -fsSL https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v2.16.1/php-cs-fixer.phar -o /usr/local/bin/php-cs-fixer
```

Comment:
@kbussell @mrj0 If you guys have a sec. Can you test these two lines?

Reply:
Could we maybe just fail with instructions how to install? This makes me nervous :)

Reply:
@porkcube @nanodocumet Your call. Both ways work for me.

Reply:
Either works but as the version is pinned and part of the URL it's not trivially abusable.

Reply:
I very open to either idea. Currently the only user I see using this is Mike. :) I don't think Keith is doing much PHP dev. :)))))))

Reply:
Docker works for me - I don't have a vm to worry about, but this doesn't require me to download things of the interwebs and run them.

```
mrj@darty:~/Projects/tradesy/core/tradesy-common$ time docker run -i --rm -v `pwd`:/opt/project gcr.io/tradesy-playground/php-linter:latest fix "$line" --rules=@PSR2 --allow-risky=no
Loaded config default.
Using cache file ".php_cs.cache".

Fixed all files in 0.098 seconds, 10.000 MB memory used

real	0m1.087s
user	0m0.047s
sys	0m0.019s
```

Reply:
I was actually planning to gut docker install since it gets very slow on macs.

Reply:
Sure, yeah thanks!